common/djangoapps/enrollment/serializers.py

```diff
@@ -36,6 +36,7 @@ class CourseSerializer(serializers.Serializer):  # pylint: disable=abstract-meth
     """
 
     course_id = serializers.CharField(source="id")
+    course_name = serializers.CharField(source="display_name_with_default")
```

Comment:
@nasthagiri said that it was preferable over `display_name` since it's possible for `display_name` to be empty.